lld/include/lld/Common/ErrorHandler.h

```diff
@@ -152,6 +152,19 @@ void message(const Twine &msg, llvm::raw_ostream &s = outs());
 void warn(const Twine &msg);
 uint64_t errorCount();
 
+enum class DiagLevel { Log, Warn, Err, Fatal };
+
+class SyncStream {
```

Comment:
I'm curious what the "Sync" part means here?    Can we come up with a more meaningful name?   Something that signifies this relates to logging/error output maybe?

Reply:
The name `SyncStream`, also used by mold, comes from https://en.cppreference.com/w/cpp/io/basic_osyncstream